hbase-plugins/src/main/java/co/cask/hydrator/plugin/sink/mapreduce/HBaseTableOutputFormat.java

```diff
@@ -27,16 +40,74 @@
  */
 public class HBaseTableOutputFormat<KEY> extends TableOutputFormat<KEY> {
 
+  private static final Logger LOG = LoggerFactory.getLogger(HBaseTableOutputFormat.class);
+  public static final String DEFAULT_COL_FAMILY = "default";
+
   @Override
   public void setConf(Configuration otherConf) {
     // To resolve CDAP-12731, set the current class's classloader to the thread's context classloader,
     // so that it gets picked up when the super.setConf() calls HBaseConfiguration.create(Configuration)
     ClassLoader originalClassLoader = Thread.currentThread().getContextClassLoader();
     Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
+    HBaseAdmin admin = null;
     try {
       super.setConf(otherConf);
+      Configuration configuration = super.getConf();
```

Comment:
is it different from `otherConf` ??

Reply:
yes it have some more configs then other configs which is added by super class.